internal/controllers/device_data_controller.go

```diff
@@ -230,6 +217,7 @@ func (d *DeviceDataController) getHistoryV1(c *fiber.Ctx, userDevice *grpc.UserD
 		},
 		Size:    some.Int(1000),
 		Source_: &source,
+		Sort:    []types.SortCombinations{types.SortOptions{SortOptions: map[string]types.FieldSort{"data.timestamp": {Order: &sortorder.Asc}}}},
```

Comment:
I think we will also need to add types.SortCombination for `_score ` to this slice so we first get random values and then sort those random values by timestamp. I am unsure if the `_score` sort would need to go before or after.

Reply:
I don't see how we're going to get both (i) random elements (ii) sorted by time in pure Elastic. Given that the existing endpoint doesn't sort by time, maybe it's okay?

Reply:
Ah, you are right. I was thinking we would just add multiple sort parameters to the sort slice, but that just means values with matching scores would be sorted by timestamp. I agree if we were not sorting them in the past then we should be fine with keeping that behavior.

Reply:
removed from this pr